demonstrations/tutorial_eqnn_force_field.py

```diff
@@ -317,7 +317,7 @@ def vqlm(data, params):
 
     # Initial encoding
     for i in range(num_qubits):
-        equivariant_encoding(alphas[i, 0], jnp.asarray(data)[i % active_atoms, ...], wires=[i])
+        equivariant_encoding(alphas[i, 0], jnp.asarray(data, dtype=complex)[i % active_atoms, ...], wires=[i])
```

Comment:
These two changes lead to horizontal scrollbars, unfortunately...

```suggestion
        equivariant_encoding(
            alphas[i, 0], jnp.asarray(data, dtype=complex)[i % active_atoms, ...], wires=[i]
        )
```